stable/postgresql/templates/_helpers.tpl

```diff
@@ -14,3 +14,14 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
```

Comment:
Yeah, I wasn't really sure which was more desirable, @prydonius and I briefly discussed the helper as a possible implementation... Though I could've easily misunderstood what he meant. 

I can unroll it.